package.json

```
@@ -24,6 +24,7 @@
     "test": "test"
   },
   "scripts": {
+    "changelog": "conventional-changelog -p angular",
```

Should be the same as https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/standard-changelog

```suggestion
    "changelog": "standard-changelog",
```

---

Well, it's actually NOT the same. `standard-changelog` output to CHANGELOG.md, which we don't really want.

---

I see. Ignore my comment then.